src/ServerPrivate.cc

```diff
@@ -246,8 +244,6 @@ void ServerPrivate::AddRecordPlugin(const ServerConfig &_config)
       recordPluginElem->Get<std::string>("compress_path", "");
     std::tie(sdfRecordResources, hasRecordResources) =
       recordPluginElem->Get<bool>("record_resources", false);
-    std::tie(sdfCompress, hasCompress) =
-      recordPluginElem->Get<bool>("compress", false);
```

Comment:
Based on what I can tell, it looks like the variables `sdfCompress` and `hasCompress` aren't used. Is that why you're removing them?

Reply:
Yup, it came up on codecheck while I was cleaning up the rest, so I thought it was small enough to sneak into this PR.